src/test/e2e/api/admin/tags.e2e.test.ts

```diff
@@ -219,3 +223,48 @@ test('backward compatibility: the API should return invalid tag names if they ex
     const { body } = await app.request.get('/api/admin/tags').expect(200);
     expect(body.tags).toContainEqual(tag);
 });
+
+test('should include tag color information when getting feature tags', async () => {
+    const featureName = 'test.feature.with.color';
+    const tag = {
+        value: 'TeamRed',
+        type: 'simple',
+    };
+
+    await app.request.post('/api/admin/projects/default/features').send({
+        name: featureName,
+        type: 'kill-switch',
+        enabled: true,
+        strategies: [{ name: 'default' }],
+    });
+
+    await app.request
+        .put('/api/admin/tag-types/simple')
```

Comment:
it's weird that we parametrize ${featureName} in the next call but hardcode simple as a string. I'd either duplicate string literals or parametrize everything